src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -363,6 +371,41 @@ def get_command(self, additional_args=None):
     # list" and then run that fuzzer.
     return self.ssh_command('ls')
 
+  def fetch_and_process_logs_and_crash(self):
+    """Fetch symbolized logs and crashes."""
+    # Get the crash from the device.
+    self.device.fetch(
+        self.fuzzer.data_path('crash*'), self.fuzzer.results_output())
+
+    for log in os.listdir(self.fuzzer.results_output()):
+      if log.startswith('fuzz-0.log'):
+        self.device.dlog(self.fuzzer.results_output(log))
+
+    # Clusterfuzz assumes that the Libfuzzer output points to an absolute path,
+    # where it can find the crash file.
+    # This doesn't work in our case due to how Fuchsia is run.
+    # So, we make a new file, change the appropriate line with a regex to point
+    # to the true location. Apologies for the hackery.
+    crash_testcase_file_path = None
+    for outfile in os.listdir(self.fuzzer.results_output()):
+      if os.path.isfile(os.path.join(self.fuzzer.results_output(),
+                                     outfile)) and 'crash-' in outfile:
```

Comment:
there's also at least 'timeout-' and 'oom-'. a better way may be to parse this from the logs (i.e. do lines 395... first) and then use that to copy the right file.

Reply:
rejiggered it around in the way suggested, lmk what you think?